cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -329,6 +329,14 @@
       + "replica distribution from each topic. For example, 1.80 means the highest topic replica count of a broker "
       + "should not be above 1.80x of average replica count of all brokers for the same topic.";
 
+  /**
+   * <code>goal.violation.distribution.threshold.multiplier</code>
+   */
+  public static final String GOAL_VIOLATION_DISTRIBUTION_THRESHOLD_MULTIPLIER_CONFIG = "goal.violation.distribution.threshold.multiplier";
+  private static final String GOAL_VIOLATION_DISTRIBUTION_THRESHOLD_MULTIPLIER_DOC = "The multiplier applied to the threshold"
+      + " of distribution goals used for detecting and fixing anomalies. For example, 2.50 means the threshold for each "
+      + "distribution goal will be 2.50x of the value used in manual goal optimization requests (e.g. rebalance).";
```

Comment:
I am thinking how we should use this multiplier. 
For example, one resource's distribution threshold is set to 1.2 and multiplier is set to 2.5(the exemplary value in current `cruisecontrol.properties` , through it is not used), 
then based on current way of calculation, the this resource's anomaly detection threshold should be 1.2*2.5 =3, which result in upper limit to be 3 and lower limit to be 0.
but there is another way to calculate, which is  (1.2 - 1)* 2.5 +1 = 1.5, which make upper limit to be 1.5 and lower limit to be 0.5
I am wondering which is better, I feel the current way of applying multiplier will greatly increase the anomaly detection/fixing threshold.

Reply:
Users, in either case, have control over by how much to increase / decrease this multiplier to adjust their limits to achieve exactly the same outcome. I think it is better to keep this computation straightforward rather than adding complexity.